podcasts/End of Year/Views/StoriesView.swift

```diff
@@ -97,28 +97,30 @@ struct StoriesView: View {
 
             VStack(spacing: 15) {
                 let progress = syncProgressModel.progress
-                CircularProgressView(value: progress, stroke: Color.white, strokeWidth: 6)
+                CircularProgressView(value: progress, stroke: Color.black, strokeWidth: 6)
                     .frame(width: 40, height: 40)
                 Text(L10n.loading)
-                    .foregroundColor(.white)
+                    .foregroundColor(.black)
                     .font(style: .body)
             }
 
             storySwitcher
```

Comment:
@bjtitus  I'm not sure if this needs to be here. Forcing the loading view or error to be in the body causes a crash when tapping. WDYT?